package.json

```diff
@@ -59,6 +59,7 @@
     "copy-to-clipboard": "^3.3.1",
     "core-js": "^3.16.3",
     "cross-fetch": "^3.1.5",
+    "csv-parse": "^5.3.6",
```

Comment:
suggest installing this only as a devDependency instead of a dependency since we're only using it for our e2e tests